packages/docs/src/scripts/components/Header.jsx

```diff
@@ -1,22 +1,110 @@
 import GitHubLinks from './GitHubLinks';
 import React from 'react';
 
-const Header = () => {
-  const rootPath = process.env.root ? `/${process.env.root}` : '/';
+class Header extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = { isBannerOpen: false };
+    this.handleToggleBannerButton = this.handleToggleBannerButton.bind(this);
+  }
 
-  return (
-    <header className="ds-base--inverse ds-u-padding--3 ds-u-display--flex ds-u-justify-content--between ds-u-align-items--center">
-      <h1 className="ds-h3 ds-u-margin-bottom--0">
-        <a href={rootPath} className="c-header__title" title="Home">
-          CMS Design System
-        </a>
-      </h1>
-      <GitHubLinks
-        className="ds-u-display--none ds-u-md-display--block"
-        inverse
-      />
-    </header>
-  );
-};
+  handleToggleBannerButton() {
+    this.setState({ isBannerOpen: !this.state.isBannerOpen });
+  }
+
+  render() {
+    const rootPath = process.env.root ? `/${process.env.root}` : '/';
+
+    const bannerHeader = this.state.isBannerOpen
+      ? 'usa-banner__header usa-banner__header--expanded'
+      : 'usa-banner__header';
+    const bannerHidden = !this.state.isBannerOpen;
+
+    return (
+      <div>
+        <div className="usa-banner">
+          <div className="usa-accordion">
+            <header className={bannerHeader}>
+              <div className="usa-banner__inner">
+                <div className="grid-col-auto">
+                  <img
+                    className="usa-banner__header-flag"
+                    src="/public/images/us_flag_small.png"
+                    alt="U.S. flag"
+                  />
+                </div>
+                <div className="grid-col-fill tablet:grid-col-auto">
+                  <p className="usa-banner__header-text">
+                    An official website of the United States government
+                  </p>
+                </div>
+                <button
+                  onClick={this.handleToggleBannerButton}
+                  className="usa-accordion__button usa-banner__button"
+                  aria-expanded="false"
```

Comment:
Shouldn't this be dynamic too like this? `aria-expanded={this.state.isBannerOpen}`